scripts/build.py

```diff
@@ -179,6 +189,27 @@ def generate_app_yaml(deploy_mode=False):
         prod_yaml_file.write(content)
 
 
+def modify_constants(prod_env, maintenance_mode):
+    """Modify constants.ts and feconf.py.
+
+    Args:
+        prod_env: bool. Whether the server is started in prod mode.
+        maintenance_mode: bool. Whether the site should be put into
+            the maintenance mode.
+    """
+    dev_mode_variable = (
+        '"DEV_MODE": false' if prod_env else '"DEV_MODE": true')
+    common.inplace_replace_file(
+        common.CONSTANTS_PATH, r'"DEV_MODE": .*', dev_mode_variable)
+
+    enable_maintenance_mode_variable = (
+        'ENABLE_MAINTENANCE_MODE = ' + python_utils.UNICODE(maintenance_mode))
```

Comment:
Please use '%s' interpolation rather than concatenation, for consistency (here and elsewhere).